src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/IdentifiersShouldNotContainUnderscoresTests.cs

```diff
@@ -19,7 +19,7 @@ public class IdentifiersShouldNotContainUnderscoresTests
     {
         #region CSharp Tests
         [Fact]
-        public async Task CA1707_ForAssembly_CSharp()
+        public async Task CA1707_ForAssembly_CSharp() // TODO: How to test the code fixer for this?
```

Comment:
I have made a couple of tests and I couldn't find anything to access the projects/assemblies afterward.

Tagging @sharwell as he might have some input/ideas.

Reply:
Ping @sharwell

Reply:
I don't know if there is an easy way to do this - also I am not sure how anyone will invoke this fix...

Reply:
@Evangelink I believe there is a feature request in roslyn to apply codefixes from command-line. When it's implemented, there should be a way to apply this codefix.